<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrientationPage</class>
 <widget class="QWidget" name="OrientationPage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_orientation_page">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Orientations</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_orientation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Orientation(s)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_orientation_single_spinbox">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_orientation_1">
          <property name="text">
           <string>Single orientation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_orientation_2">
          <property name="text">
           <string>alpha(deg)=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_orientation_alpha"/>
        </item>
        <item>
         <widget class="QLabel" name="label_orientation_3">
          <property name="text">
           <string>beta(deg)=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_orientation_beta"/>
        </item>
        <item>
         <widget class="QLabel" name="label_orientation_4">
          <property name="text">
           <string>gamma(deg)=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_orientation_gamma"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_orientation_single_linedit">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_orientation_5">
          <property name="text">
           <string>Single orientation </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_orientation_single">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>(format: &quot;alpha,beta,gamma&quot; (deg)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_orientation_preset">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_orientation_6">
          <property name="text">
           <string>Preset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_orientationpresets">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Level:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_orientation_level">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_orientation_values">
          <property name="text">
           <string>(values(0))</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioButton_orientation_from_file">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_orientation_7">
          <property name="text">
           <string>File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_orientation_file">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_orientation_preset</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_orientationpresets</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_orientation_from_file</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_orientation_file</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_orientation_single_linedit</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_orientation_single</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_orientation_from_file</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_orientation_single_spinbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_orientation_alpha</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_orientation_single_spinbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_orientation_beta</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_orientation_single_spinbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_orientation_gamma</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
